### PR TITLE
[react/axios] add h2Service

### PR DESCRIPTION
**Pull request by dgbirm**

### CODE REVIEW

File: src/main/java/com/habibank/controller/HabibankController.java [hunk 1]
```diff
@@ -90,14 +88,15 @@ private String index() {
...
+//create customer	
+	@PostMapping("api/customers")
+	private ResponseEntity<?> createEmp(@RequestBody Customer c) {
+		Customer createdCustomer = new Customer(
+				c.getUserName(),c.getFullName(),c.getEmail(),
+				c.getAddress(),c.getPhoneNumber());
+		this.custRepo.saveAndFlush(createdCustomer);
+		URI location = ServletUriComponentsBuilder.fromCurrentRequest().path("/{id}").buildAndExpand(createdCustomer.getCustomerID()).toUri();
+		return ResponseEntity.created(location).build();
+	}
```
Comment (ChrisJabb21) [inline src/main/java/com/habibank/controller/HabibankController.java]: This all looks good to me. Good job, I would say just _change createEmp to createCust_ on naming.  I was just stuck and thinking whether we should include the password in the post creation for the validation side of the API React side and Spring boot side.  This does look good for testing

File: src/main/java/com/habibank/controller/HabibankController.java [hunk 2]
```diff
@@ -90,14 +88,15 @@ private String index() {
...
-//	private ResponseEntity<?> createEmp(@RequestBody Account e) {
-//		Account createdAccount = new Account(
-//				e.getFullName(),e.getDep(),e.getJobTitle(), e.getYearlySalary());
-//		this.acctRepo.saveAndFlush(createdAccount);
-//		URI location = ServletUriComponentsBuilder.fromCurrentRequest().path("/{id}").buildAndExpand(createdAccount.getEmpID()).toUri();
-//		return ResponseEntity.created(location).build();
-//	}
+//create customer	
+	@PostMapping("api/customers")
+	private ResponseEntity<?> createEmp(@RequestBody Customer c) {
```
Comment (ChrisJabb21) [inline src/main/java/com/habibank/controller/HabibankController.java]: Just a quick naming convention change to createCust

**Review by ChrisJabb21 — APPROVED** [review]

File: src/main/java/com/habibank/controller/HabibankController.java (hunk 1 above)
Comment (dgbirm) [reply src/main/java/com/habibank/controller/HabibankController.java]: true thats a vestige from some other code

Comment (ChrisJabb21) [reply src/main/java/com/habibank/controller/HabibankController.java]: It is all good, it happens to the best of us

**Pull request merged**